### ISSUE TITLE
2Captcha Chrome extension is not working for FunCaptcha(Arkose Labs)

### ISSUE THREAD

**Issue by honest888** [2captcha/2captcha-javascript#17]
Hello, how are you?
I will be happy why your 2captcha chrome extension is not working for Arkose Labs captcha. 
https://demo.arkoselabs.com/?key=DF9C4D87-CB7B-4062-9FEB-BADB6ADA61E6
This is demo URL. 

I have checked 2Captcha detector. 
That detected the FunCaptcha. But not working.
Balance of account is fine.

**Comment by braniubojni** [comment]
Yup, same here, takes a bit long time to solve but after it nothing is happening
![image](https://github.com/user-attachments/assets/53f3612b-7d1f-47c0-8c15-e312f0725a6d)

**Comment by dzmitry-duboyski** [comment]
Hi @honest888 and @braniubojni,
Please [create a new ticket](https://2captcha.com/support/tickets/new) with the problem description. In the ticket, include the page where you're facing issues using the extension, the version of the extension, and the browser you're using.

Thanks!

**Comment by braniubojni** [comment]
> Hi @honest888 and @braniubojni, Please [create a new ticket](https://2captcha.com/support/tickets/new) with the problem description. In the ticket, include the page where you're facing issues using the extension, the version of the extension, and the browser you're using.
> 
> Thanks!

Hello @dzmitry-duboyski thank you for your guid. Ticket is created, ticket id is #5523.
Let me know if I did something wrong
